src/Sulu/Bundle/PageBundle/Resources/js/views/PageList/PageList.js

```diff
@@ -39,11 +39,12 @@ class PageList extends React.Component<Props> {
     static getDerivedRouteAttributes(route: Route, attributes: AttributeMap) {
         return {
             active: ListStore.getActiveSetting(PAGES_RESOURCE_KEY, getUserSettingsKeyForWebspace(attributes.webspace)),
+            locale: userStore.contentLocale,
         };
     }
 
     @action setDefaultLocaleForWebspace = () => {
```

Comment:
Since the function should not set anything anymore, I would rename it to something like `redirectToWebspaceLocale`. Other suggestions are welcome :slightly_smiling_face: